src/lib/tuning/index/column_ref.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * A ColumnRef is used to reference a set of columns by a table name
+ * and their column indexes in this table.
+ * This is used by the IndexEvaluator, IndexChoices and IndexOperations.
+ */
+struct ColumnRef {
+  ColumnRef(std::string table_name, ColumnID column_id);
+  ColumnRef(std::string table_name, const std::vector<ColumnID>& column_ids);
+
+  std::string table_name;
```

Comment:
Any reason to hold the table‘s name instead of a pointer?

Reply:
Is there a way to determine the table_name if we only have a pointer to it? We need this only for our logs, but meaningful logs seem important enough. If we can not get the table name from its pointer we propose sticking with the name, as we can not see any real drawbacks in using it.